examples/nidaqmx_analog_input/_helpers.py

```diff
@@ -300,3 +301,12 @@ def get_sessions_and_channels_for_pins(
         raise KeyError(f"Pin(s) {pins} and site {site} not found")
 
     return session_and_channel_info
+
+
+def generate_wrapper_enum(
+    enum_type: Enum
+) -> Enum:
+    wrapper_enum_dict = {k:v.value for k,v in enum_type.__members__.items()}
```

Comment:
>  `_helpers.py:309: error: "Enum" has no attribute "__members__"  [attr-defined]`

https://github.com/python/typeshed/blob/main/stdlib/enum.pyi has had `EnumMeta.__members__` since 2/22/2022.

I think this is failing because `enum_type: Enum` declares it to be an instance of `Enum`, not a type derived from `Enum`. Try `enum_type: Type[Enum]` and `-> Type[Enum]`

Reply:
And yeah, mypy can't handle enums with dynamic members.
https://mypy-play.net/?mypy=latest&python=3.11&gist=63649673382c32dfd9d77b912040997a
![image](https://github.com/ni/measurementlink-python/assets/17914399/153cd137-10b7-4cc7-a6c1-5b237f8df102)

Reply:
> `
_helpers.py:302: error: Incompatible return value type (got "Enum", expected "type[Enum]")  [return-value]`

Believe it or not, splitting this into `enum_type = Enum(...)` and `return enum_type` seems to prevent mypy from thinking `Enum()` returns an instance of `Enum`. But I think you will run into other errors instead.

Reply:
Yeah, I didn't have much luck in getting this to work either. I've reverted the change for dynamically creating enums.

Reply:
@WesleyTangNationalInstruments please update the PR description not to mention the dynamic helper if it's gone.